README.md

```diff
@@ -56,15 +56,15 @@ seed should be a 32 byte buffer.
 Output is in the form
 
     {
-      "curve": "ed25519",
-      "public": "<public_key_here>.ed25519",
-      "private": "<private_key_here>.ed25519",
-      "id": "@<public_key_here>"
+      "curve": "ed25519", // Our current eliptic curve
+      "public": "fHlqm8WWP4ODtsuXazhJR66YrmhOWcmqae4LxXPHn0o=.ed25519", // format: public_key.curve
+      "private": "++uy5C8c0q+RpkyDQogzYSN4py3gL3m/qCN2YKE1snp8eWqbxZY/g4O2y5drOElHrpiuaE5Zyapp7gvFc8efSg==.ed25519", // format: private_key.curve
+      "id": "@fHlqm8WWP4ODtsuXazhJR66YrmhOWcmqae4LxXPHn0o=" // format: @public_key
     }
 
 **Note:**
 
-- The public key consists of 32 random bytes base64 encoded.
+- The public key consists of 32 bytes base64 encoded.
```

Comment:
Was "random" incorrect here?

Reply:
Yep. See this post by kas:

[%UXR8mzDb5H1Kbt3ZBSyVLZv9sLXrNXDh1mKfIcx0GvY=.sha256](%UXR8mzDb5H1Kbt3ZBSyVLZv9sLXrNXDh1mKfIcx0GvY=.sha256)

Apparently, the public key can be completely recreated *using just the 32 byte **secret part*** of the secret key.

Reply:
I was wondering if we should dive in a document this as well.

Reply:
the private key encoding also includes a copy of the public key, so that it doesn't need to recalculate it. that's not obvious if you look at it in base64 encoding though, but if you convert it to hex, you'll see it.